nixos/tests/initrd-verify.secret

```diff
@@ -0,0 +1 @@
+not-secret:0f9ZmCOeRdEa5HitWkOOfQDJQcrif06V+apZpSSHJ/WZL9ZDI2Guw7qEw3/cjBtKgh1r8MJoqQPWUmkL4gjZ5g==
```

Comment:
To fix the editorconfig check we probably need to add a line sort of like https://github.com/NixOS/nixpkgs/blob/28089f8874ffdd06c52dc3b8e94dccf6ac8fffa8/.editorconfig#L82